test/integration/builder/unions.js

```diff
@@ -112,4 +114,220 @@ module.exports = function(knex) {
         );
     });
   });
+
+  if (
+    ['pg', 'mssql', 'pg-redshift', 'oracledb', 'sqlite3'].includes(
+      knex.client.driverName
+    )
+  ) {
+    describe.only('intersects', function() {
+      before(function() {
+        return knex.schema.createTable('intersect_test', function(t) {
+          t.integer('id');
+          t.integer('test_col_1');
+          t.integer('test_col_2');
+          t.integer('test_col_3');
+        });
+      });
+
+      beforeEach(function() {
+        return knex('intersect_test').insert([
+          {
+            id: 1,
+            test_col_1: 1,
+            test_col_2: 2,
+            test_col_3: 1,
+          },
+          {
+            id: 2,
+            test_col_1: 2,
+            test_col_2: 3,
+            test_col_3: 1,
+          },
+          {
+            id: 3,
+            test_col_1: 2,
+            test_col_2: 3,
+            test_col_3: 2,
+          },
+          {
+            id: 4,
+            test_col_1: 1,
+            test_col_2: 2,
+            test_col_3: 2,
+          },
+          {
+            id: 5,
+            test_col_1: 1,
+            test_col_2: 2,
+            test_col_3: 1,
+          },
+        ]);
+      });
+
+      after(function() {
+        return knex.schema.dropTableIfExists('intersect_test');
```

Comment:
dropTableIfExists is unnecessary here, you can just drop it always.

Reply:
Sure thing